run-dosbox

```diff
@@ -103,6 +103,21 @@ def setup_midi_game():
         rfile.apply_rpatch(STEAM_APP_ID, 'midi_' + x)
 
 
+def setup_bundle(distdir):
+    extend_env = (
+        ('PATH', os.path.join(distdir, 'bin')),
+        ('LD_LIBRARY_PATH', os.path.join(distdir, 'lib')),
+    )
+    for env_var, path in extend_env:
+        if os.path.isdir(path):
+            sys_path_str = os.getenv(env_var, None)
+            sys_path = sys_path_str.split(os.pathsep) if sys_path_str else []
+            if dir not in sys_path:
```

Comment:
I believe this should be: `if path not in sys_path`?

Reply:
Sure.